backups/backup.py

```diff
@@ -105,20 +107,30 @@ def run_importer_script() -> None:
                 )
     finally:
         if popen:
+            stderr = popen.stderr.read()
             popen.terminate()
 
     if (return_code := popen.returncode) != 0:
-        raise subprocess.CalledProcessError(return_code, command)
-
-    if not re.search(r"[1-9][0-9]* projects imports succeeded", stdout):
+        log.exception("Command failed with stderr: '%s'", stderr)
         raise RuntimeError(
-            f"{ERROR_PREFIX} - '[1-9][0-9]* projects imports succeeded' not found in logs"
-        )
-    if not re.search(r"[1-9][0-9]* copies succeeded", stdout):
-        raise RuntimeError(
-            f"{ERROR_PREFIX} - '[1-9][0-9]* copies succeeded' not found in logs"
+            f"Capella importer failed with exit code {return_code}"
         )
 
+    if is_capella_5_x_x():
```

Comment:
It seems like only capella 5.0.0 has different success messages. Therefore, the tests currently fail since we also go into the if case for 5.2.0.